### PR TITLE
Add token_overlap config option

### PR DESCRIPTION
**Pull request by samleegithub**
Add token_overlap option in config file.

### CODE REVIEW

**Comment by sarah-widder** [comment]
Hi @samleegithub thanks for contributing! Looks good to me. If you want to resolve the conflict I'll come back and approve.

File: scripts/config.json [hunk 1]
```diff
@@ -7,6 +7,7 @@
         "search_service_name": "<search service name to use or create>",
         "index_name": "<index name to use or create>",
         "chunk_size": 1024,
+        "token_overlap": 128,
```
Comment (sarah-widder) [inline scripts/config.json:10]: Update the example in the README with the same?

Comment (samleegithub) [reply scripts/config.json:10]: Good idea. Will do.

Comment (samleegithub) [reply scripts/config.json:10]: Done

**Review by sarah-widder — APPROVED** [review]

**Pull request merged**